### DISCUSSION TITLE
Organizing a Community

### DISCUSSION THREAD

**Discussion by judeibe** [community-scripts/ProxmoxVE#8] [General]
As this project is no longer just one person, I believe a baseline of organization is needed and as the project matures additional layers can be added.

Starting off I think moderators for the discussions and pr comments will needed. That would consist of closing issues or discussions. Actions can be used to automate some portions of this.

**Comment by MickLesk** [comment]
We have already turned the new project into an organization. If you have the motivation to get involved, you are welcome to get in touch with me and briefly explain what you want to achieve with the community project or where you would like to get involved.

**Reply by Mellowlynx** [reply]
As you don't share any contact information, getting it in touch with you in private is not possible.

**Reply by MickLesk** [reply]
I'll create a general contact email tomorrow, I'm already so busy today

**Reply by CrazyWolf13** [reply]
@MickLesk  
Maybe it would be a good idea to set up some sort of matrix channel setup?
This would make communication easier, as the focus on this repo is forming a community and decentralising, so I'd say starting with a matrix channel instead of personal email would be beneficial, however this is fully up to you!

**Reply by Mellowlynx** [reply]
@MickLesk you join https://discord.gg/U6PzaFFj, some of us are already there.

**Comment by lotsokluze** [comment]
Misunderstood.

@MickLesk is a darling of a person.

I'm an idiot 🤯

@tteck is legend.

**Reply by judeibe** [reply]
@lotsokluze I would not resort to personal attacks

@MickLesk if the there is a role I'm interested in I'll let it be known. But of interest to transparency I'll keep my discussions in the open.

**Reply by MickLesk** [reply]
What is this terrible accusation? 

My point was simply that he can write to me if he wants to contribute. I can't make everyone a member of the organization on the very first day. Especially as there's still a lot to manage at the moment.


@tteck

**Reply by Mellowlynx** [reply]
There will be a need for moderation and monitoring, opening it up for everyone, will also cause problems.
So see what happens and use that info to make an educated decision.

**Reply by lotsokluze** [reply]
Misunderstood the premise.

Sorry to @MickLesk 
Sorry to @judeibe

**Comment by newzealandpaul** [comment]
Hi @MickLesk I am interested in being involved in software development and security. Please get in touch (as there is no way for me to contact you).

I think Discord would be a good place for interactive discussion.

**Reply by judeibe** [reply]
I'm interested to help on the community side mostly. Discussion, documentation and the site.

**Comment by judeibe** [comment]
I'm closing this as I feel, the foundations are laid.
